vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/Holder.kt

```diff
@@ -182,3 +210,24 @@ interface Holder {
     }
 
 }
+
+fun Map<InputDescriptor, Map<SubjectCredentialStore.StoreEntry, Map<ConstraintField, NodeList>>>.toDefaultSubmission(): Map<String, InputDescriptorCredentialSubmission> {
+    return mapNotNull { descriptorCredentialMatches ->
```

Comment:
Please refactor to expression body

Reply:
fixed in commit https://github.com/a-sit-plus/kmm-vc-library/commit/65b06496293f5ec3ea9babe119596f8a137085d8